pkg/session/eventlisteners.go

```diff
@@ -44,7 +44,7 @@ func (sess *Session) listenFunc(ctx context.Context) func(ev interface{}) {
 				req.Initiator = ev.Initiator.Type.String()
 			}
 		case *network.EventLoadingFailed:
-			log.Debugf("Loading failed: %v, %v, Reason; %v, Cancel: %v, %v, %v", ev.RequestID, ev.Type, ev.BlockedReason, ev.Canceled, ev.ErrorText, ev.Timestamp.Time())
+			log.Tracef("Loading failed: %v, %v, Reason; %v, Cancel: %v, %v, %v", ev.RequestID, ev.Type, ev.BlockedReason, ev.Canceled, ev.ErrorText, ev.Timestamp.Time())
```

Comment:
Usikker på om det er lurt å sette denne til Trace. Selv om det fører til mye logging, så er denne informasjonen veldig verdifull for feilsøking. Vil tro det er bedre å gå gjennom de forskjellige logmeldingene og aktivt håndtere alle som vi tror bare er støy.